cellbrowser_tools/createJobsFromCSV.py

```diff
@@ -101,10 +100,8 @@ def do_image(args, prefs, rows):
 
     info.cbrThumbnailSize = 128
 
-    if not os.path.exists(info.cbrImageLocation):
-        os.makedirs(info.cbrImageLocation)
-    if not os.path.exists(info.cbrThumbnailLocation):
-        os.makedirs(info.cbrThumbnailLocation)
+    os.makedirs(info.cbrImageLocation, exist_ok=True)
+    os.makedirs(info.cbrThumbnailLocation, exist_ok=True)
```

Comment:
I ran into some surprising issues with this on some early tests because I think two dask workers hit this line at roughly the same time and they both entered. One of them created the directory and the other tried to make the same directory and it resulted in a `FileExistsError` :joy:

Reply:
that is scary

Reply:
but not surprising!